src/main/java/seedu/address/model/ReadOnlyFriendlyLink.java

```diff
@@ -14,6 +16,16 @@ public interface ReadOnlyFriendlyLink {
      * This list will not contain any duplicate persons.
      */
     ObservableList<Person> getPersonList();
+    /**
+     * Returns an unmodifiable view of the volunteer list.
+     * This list will not contain any duplicate persons.
+     */
+    ObservableList<Volunteer> getVolunteerList();
+    /**
+     * Returns an unmodifiable view of the elderly list.
+     * This list will not contain any duplicate persons.
+     */
+    ObservableList<Elderly> getElderlyList();
```

Comment:
Refer to `ReadOnlyElderly` and `ReadOnlyVolunteer` 😅 I have added these methods.

Reply:
I need to add it though. There's the `resetFriendlyLinkData` method in `FriendlyLink` that's currently not setting the elderly and volunteers. Another way we can do this is have `ReadOnlyFriendlyLink` implement `ReadOnlyVolunteer` and `ReadOnlyElderly`